src/main/java/com/akhil/app/controllers/HealthController.java

```diff
@@ -0,0 +1,20 @@
+package com.akhil.app.controllers;
+
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HealthController {
+
+    @GetMapping("/healthup")
```

Comment:
/health is enough,
It should send status:Up if database server or any other services are working fine
It should send status:Down 
If database servers are down

As we don't have db servers here you can send status:Up directly

Reply:
Made the changes and committed again, please check